[1m[31mERROR   [0m root:test_get_user_details_all.py:51 Response is --> [{'id': 5543039, 'name': 'Divakar Marar', 'email': 'marar_divakar@christiansen-rosenbaum.example', 'gender': 'male', 'status': 'active'}, {'id': 5543038, 'name': 'Ganak Shah', 'email': 'shah_ganak@jaskolski-price.test', 'gender': 'male', 'status': 'active'}, {'id': 5543036, 'name': 'Brijesh Dwivedi', 'email': 'brijesh_dwivedi@prosacco-haley.example', 'gender': 'female', 'status': 'active'}, {'id': 5543035, 'name': 'Meghnath Talwar', 'email': 'talwar_meghnath@hessel.test', 'gender': 'female', 'status': 'active'}, {'id': 5543033, 'name': 'Daiwik Bandopadhyay', 'email': 'bandopadhyay_daiwik@schuppe.example', 'gender': 'male', 'status': 'inactive'}, {'id': 5543032, 'name': 'Harinakshi Desai DO', 'email': 'do_harinakshi_desai@okuneva.example', 'gender': 'male', 'status': 'active'}, {'id': 5543030, 'name': 'Msgr. Dayamayee Marar', 'email': 'marar_dayamayee_msgr@upton.example', 'gender': 'female', 'status': 'active'}, {'id': 5543029, 'name': 'Purnima Somayaji', 'email': 'somayaji_purnima@hamill-nicolas.example', 'gender': 'female', 'status': 'inactive'}, {'id': 5543028, 'name': 'Agnivesh Asan', 'email': 'agnivesh_asan@mclaughlin.example', 'gender': 'female', 'status': 'inactive'}, {'id': 5543027, 'name': 'Tej Arora', 'email': 'tej_arora@johnston-buckridge.test', 'gender': 'male', 'status': 'active'}]
[1m[31mERROR   [0m root:test_get_user_details_all.py:52 Actual Status code is --> 200 but expected is 401, that's why it's failed